d3a_api_client/setups/test_create_aggregator.py

```diff
@@ -28,16 +33,18 @@ def on_market_cycle(self, market_info):
             return
 
         for device_event in market_info["content"]:
-            if "available_energy_kWh" in device_event["device_info"] and \
-                    device_event["device_info"]["available_energy_kWh"] > 0.0:
-                self.add_to_batch_commands.offer_energy(device_event["area_uuid"], price=1,
-                                                        energy=device_event["device_info"]["available_energy_kWh"] / 2)
+            if key_in_dict_and_not_none_and_greater_than_zero(
+                    device_event, "available_energy_kWh"):
```

Comment:
Single quotes vs double quotes :P

Reply:
Single quotes were also not addressed in the changes. Let's keep it in mind for the next PRs :)